packages/wallets/localize-loader.js

```diff
@@ -31,10 +38,16 @@ module.exports = async function (source) {
                                 key
                             )} and ${value}.`
                         );
-                    path.node.arguments[0] = {
-                        type: 'StringLiteral',
-                        value: key,
-                    };
+
+                    // This ensures that if no translations have been translated for the strings yet, don't transpile the strings into keys
+                    // otherwise, i18n will display the keys instead of the strings
+                    // only transpile when translations for the strings are available
+                    if (defaultMessages.has(key)) {
+                        path.node.arguments[0] = {
+                            type: 'StringLiteral',
+                            value: key,
+                        };
+                    }
```

Comment:
i.e.:
```
{i18n.t('Deposit')}
```
The loader will transpile it to the key itself `0da00b60` this in build time:
```
{i18n.t('0da00b60')}
```
If `0da00b60` does not exist in the translations JSON files, i18n will show that key in the browser itself! So we don't want that, thus we just don't transpile the strings to keys in this case